proxy/static/index.html

```diff
@@ -1,97 +1,117 @@
 <!doctype html>
-<html lang="en">
+<html>
   <head>
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1" />
     <title>Snowflake</title>
-    <link rel="icon" href="assets/favicon.ico" />
-    <link rel="stylesheet" href="index.css" />
+    <link rel="icon" href="./assets/favicon.ico" />
+    <link rel="stylesheet" href="./bootstrap.css" />
```

Comment:
Nitpick: Bootstrap's CSS file is 176 KB in size but I suspect we only need a small subset of its code for the language switcher. Should we cherry-pick what we really need?

Reply:
Fixed. It's much smaller now, and I removed some unnecessary styles from index.html